test/aevm_chain_SUITE.erl

```diff
@@ -107,6 +124,7 @@ contracts(_Cfg) ->
     {[_Acc, _Acc2, Contract1, Contract2], S} = setup_chain(),
     _S1 = lists:foldl(fun({Value, Arg}, S0) -> make_call(Contract1, Contract2, Value, Arg, S0) end,
                       S, [{(I - 3) * 100, I + 100} || I <- lists:seq(1, 10)]),
+    teardown_chain(),
```

Comment:
If test fails, leftover mock may affect following tests? It may benefit from end_per_testcase.

Reply:
Actually the whole mocking thing is not needed any more...